python/dgl/distributed/__init__.py

```diff
@@ -1,8 +1,9 @@
 """DGL distributed."""
 
-from .dist_graph import DistGraphServer, DistGraph, node_split, edge_split
+from .dist_graph import DistGraphServer, DistGraph, DistTensor, node_split, edge_split
 from .partition import partition_graph, load_partition, load_partition_book
 from .graph_partition_book import GraphPartitionBook, RangePartitionBook, PartitionPolicy
+from .sparse_emb import SparseAdagrad, SparseNodeEmbedding
```

Comment:
SparseNodeEmbedding can be SparseEmbedding. There is no difference between SparseNodeEmbedding and SparseEdgeEmbedding.

Reply:
there is. It gets a graph as input and creates embeddings for nodes.